### DISCUSSION TITLE
[Kavita] Idea / Scan result

### DISCUSSION THREAD

**Discussion by Jayknightfr** [Kareadita/Kavita#3004] [Ideas]
### Idea Description

So first of all i love Kavita. But i've been bugged by the fact that when i add new material, scan, and don't get any feedback. And when i don't see my new material, i have to dig through the logs to see what happened. Sometimes it's a missing tag in a epub file, sometimes another issue. Sometimes everything works well but still no feedback on when the scan is over, etc.

I'd love to have some info about the last scan : number of discovered items, updated items, problems with files. Also the ability to see the logs directly from Kavita would be great when there's an issue.

Thanks!

### Idea Category

User Experience

### Duration of Using Kavita

1 year

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by scare376** [comment]
Not trying to discredit the idea but there are ways to see this info. After a scan new items will appear in the newly added and newly updated series sections on the home page. If there is a problem with a book and Kavita couldn't import it it should appear in the media tab of the admin dashboard under media issues.

**Reply by scare376** [reply]
Also while the scan is in progress the icon in the upper right well be green and tell you where it's at in the scan when you click it

**Reply by Cducharme84** [reply]
> Also while the scan is in progress the icon in the upper right well be green and tell you where it's at in the scan when you click it

Next stable will have a series left to process counter here too.

**Reply by Jayknightfr** [reply]
> Also while the scan is in progress the icon in the upper right well be green and tell you where it's at in the scan when you click it

Yeah my bad it scanned so fast i didn't have time to realize that.

I guess we don't need all of that, i also didn't know about the media tab so thanks!
